docs/spark/how-to-guides/udf-guide.md

```diff
@@ -180,6 +182,12 @@ Notice that `func` and `func2` no longer share a closure, and they have their ow
 * Null values in UDFs can throw exceptions. It's the responsibility of the developer to handle them.
 * UDFs don't leverage the optimizations provided by Spark's built-in functions, so it's recommended to use built-in functions where possible.
 
+## FAQs
+
+1. Why do I get the error `System.NotImplementedException: The method or operation is not implemented.` or `System.InvalidCastException: Unable to cast object of type 'System.Collections.Hashtable' to type 'System.Collections.Generic.IDictionary` when trying to call a UDF with `ArrayType`, `MapType`, `ArrayList`, or `HashTable` as argument or return type?  
```

Comment:
Since there's only one at the moment, maybe it might not make sense to make it a numbered list.

Also, can we bolden the question so that it's easier to see what the question and answer is.

Reply:
Done.